### DISCUSSION TITLE
b5打开之后闪退

### DISCUSSION THREAD

**Discussion by Kingbirdzmr** [cangzhang/ChampR#282] [General]
如题

**Comment by Kingbirdzmr** [comment]
![image](https://github.com/cangzhang/champ-r/assets/43699171/5f8e04e7-a127-49ab-be1a-051cbafab023)

**Reply by cangzhang** [reply]
先登录 LoL 再打开 ChampR 可临时规避该问题，晚点发新版本

**Comment by cangzhang** [comment]
https://github.com/cangzhang/champ-r/releases/tag/v2.0.2-b6
